### DISCUSSION TITLE
Attempting to download iTunes onto a Chromebook...

### DISCUSSION THREAD

**Discussion by mriske01** [microsoft/WSL#9150] [General]
So i'm using the correct codes to download iTunes and for some reason its not working. I've done everything else before it yet it still says 
/usr/bin/wine: 40: exec: /usr/lib/wine/wine: Exec format error

help?!

**Comment by NotTheDr01ds** [comment]
Apologies if I'm misreading your question, but I think you might be in the wroooooooong place ;-).

This is the Github repo for the Windows Subsystem for Linux, a system that allows you to run *Linux* applications under *Windows 10 or 11*.

You seem to be using [Wine](https://www.winehq.org), which does the *opposite* -- It allows you to run *Windows* applications under *Linux*.
